src/atomicui/organisms/SearchBox/SearchBox.tsx

```diff
@@ -339,12 +348,49 @@ const SearchBox: React.FC<SearchBoxProps> = ({
 									))}
 								</Flex>
 							),
-							Empty:
+							Empty: !isNLPChecked ? (
 								!!value && !suggestions?.length ? (
 									<Flex className="not-found-container">
 										<NotFoundCard />
 									</Flex>
 								) : null
+							) : (
+								<Flex
+									gap={0}
+									alignItems="start"
+									className="inner-search-component"
+									// marginLeft="10px"
+									style={{
+										flexDirection: "column",
+										gap: "0"
+									}}
+								>
+									<h3>Try Asking:</h3>
+									<Flex
+										style={{
+											flexDirection: "column",
+											gap: "0.1px",
+											marginBottom: "12px",
```

Comment:
Seems excessive

Reply:
spacing becomes too large or small if I change the margin or the gap